From eb3c72486712598bc965f5cd2eec25912e71e56f Mon Sep 17 00:00:00 2001
From: Mahak Swami <mahak.swami@nxp.com>
Date: Tue, 23 Feb 2016 23:19:24 +0530
Subject: [PATCH 1055/1383] Changing syntax to #include in dts file for usdpaa
 for ls1043 for SDK2.0

[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
index 77633a5..d46ee6a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
@@ -8,7 +8,7 @@
  * warranty of any kind, whether express or implied.
  */
 
-/include/ "fsl-ls1043a-rdb.dts"
+#include "fsl-ls1043a-rdb.dts"
 
 / {
 	bp7: buffer-pool@7 {
-- 
2.8.1

